examples/yolov4.cpp

```diff
@@ -15,12 +15,22 @@
 #include "net.h"
 
 #include <opencv2/core/core.hpp>
+#include <opencv2/videoio/videoio.hpp>
 #include <opencv2/highgui/highgui.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
-#include <stdio.h>
+
 #include <vector>
+#include <iostream>
+
+#include <string.h>
+#include <stdio.h>
 
-#define YOLOV4_TINY 1 //0 or undef for yolov4
+#define NCNN_PROFILING
+#define YOLOV4_TINY //Using yolov4_tiny, if undef, using original yolov4
+
+#ifdef NCNN_PROFILING
+#include <chrono>
```

Comment:
ncnn benchmark.h has the necessary timestamp functions
Besides, c++11 is not mandatory when building with opencv 2.x